scrambles/src/main/java/net/gnehzr/tnoodle/puzzle/MegaminxPuzzle.java

```diff
@@ -83,7 +80,8 @@ public Dimension getPreferredSize() {
     private static final double UNFOLDWIDTH = 4 * Math.cos(.1 * Math.PI) + 2 * Math.cos(.3 * Math.PI);
 
     private static void turn(int[][] image, Face side, int dir) {
-        dir = GwtSafeUtils.modulo(dir, 5);
+        dir = ((dir % 5) + 5) % 5;
```

Comment:
Just curious, why the change here?

Reply:
Because I wanted to get rid of GwtSafeUtils, see above.